modules/util/units.js

```diff
@@ -108,31 +108,34 @@ function clamp(x, min, max) {
 
 function displayCoordinate(deg, pos, neg) {
     var locale = localizer.localeCode();
-    var min = (Math.abs(deg) - Math.floor(Math.abs(deg))) * 60;
-    var sec = (min - Math.floor(min)) * 60;
-    var displayDegrees = t('units.arcdegrees', {
-        quantity: Math.floor(Math.abs(deg)).toLocaleString(locale)
-    });
-    var displayCoordinate;
+    var degreesFloor = Math.floor(Math.abs(deg));
+    var min = (Math.abs(deg) - degreesFloor) * 60;
+    var minFloor = Math.floor(min);
+    var sec = (min - minFloor) * 60;
 
-    if (Math.floor(sec) > 0) {
-        displayCoordinate = displayDegrees +
+    var displayCoordinate;
+    var secToFixedDisplayPrecision = 1;
+    var secFixed = Number(sec.toFixed(secToFixedDisplayPrecision));
```

Comment:
The previous code rounded the seconds, so this should similarly round the seconds to the nearest tenth instead of flooring it to the lower tenth.

Reply:
I seems tofixed is not a proper way , so I change it to Math.round,
it always not precise, ex: 0.05 after calculate became 0.04999999 , 
so i just round it two time .
I will try to write test to test this and dmsMatcher case